packages/wallet/test/util.ts

```diff
@@ -0,0 +1,8 @@
+import { SelectionConstraints } from '@cardano-sdk/cip2';
+
+export const NO_CONSTRAINTS: SelectionConstraints = {
```

Comment:
These are technically not "no constraints". There is a [similar object](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/cip2/test/RoundRobinRandomImprove.test.ts#L17) in `cip2` tests, should probably flag it to move to `dev-utils` package when we create it.

Reply:
Right, I just kept the same name to make a refactor easy

Reply:
Done in c33edc69a400bc39617f62ee2d810f7b0e54f5c2